src/Illuminate/Validation/Rules/Email.php

```diff
@@ -0,0 +1,339 @@
+     *
+     * @var array
+     */
+    protected $data;
+
+    /**
+     * The validator performing the validation.
+     *
+     * @var \Illuminate\Validation\Validator
+     */
+    protected $validator;
+
+    /**
+     * The callback that will generate the "default" version of the file rule.
+     *
+     * @var string|array|callable|null
+     */
+    public static $defaultCallback;
+
+    /**
+     * Set the default callback to be used for determining the email default rules.
+     *
+     * If no arguments are passed, the default email rule configuration will be returned.
+     *
+     * @param  static|callable|null  $callback
+     * @return static|void
+     */
+    public static function defaults($callback = null)
+    {
+        if (is_null($callback)) {
+            return static::default();
+        }
+
+        if (! is_callable($callback) && ! $callback instanceof static) {
+            throw new InvalidArgumentException('The given callback should be callable or an instance of '.static::class);
+        }
+
+        static::$defaultCallback = $callback;
+    }
+
+    /**
+     * Get the default configuration of the file rule.
+     *
+     * @return static
+     */
+    public static function default()
+    {
+        $email = is_callable(static::$defaultCallback)
+            ? call_user_func(static::$defaultCallback)
+            : static::$defaultCallback;
+
+        return $email instanceof self ? $email : new self();
+    }
+
+    /**
+     * A strict rule set which includes DNS and Spoof checks.
+     *
+     * @return static
+     */
+    public static function strictSecurity()
+    {
+        return (new self())->strict()->dns()->spoof();
+    }
+
+    /**
+     * Validate against NoRFCWarningsValidation.
+     *
+     * @return $this
+     */
+    public function strict()
+    {
+        $this->strict = true;
+
+        return $this;
+    }
+
+    /**
+     * Validate against DNSCheckValidation.
+     * Requires the PHP intl extension.
+     *
+     * @return $this
+     */
+    public function dns()
+    {
+        $this->dns = true;
+
+        return $this;
+    }
+
+    /**
+     * Validate against SpoofCheckValidation.
+     * Requires the PHP intl extension.
+     *
+     * @return $this
+     */
+    public function spoof()
+    {
+        $this->spoof = true;
+
+        return $this;
+    }
+
+    /**
+     * Validate against FilterEmailValidation.
+     *
+     * @return $this
+     */
+    public function filter()
+    {
+        $this->filter = true;
+
+        return $this;
+    }
+
+    /**
+     * Validate against FilterEmailValidation::unicode().
+     *
+     * @return $this
+     */
+    public function filterUnicode()
+    {
+        $this->filter_unicode = true;
+
+        return $this;
+    }
+
+    /**
+     * Validate against RFCValidation.
+     *
+     * @return $this
+     */
+    public function rfc()
+    {
+        $this->rfc = true;
+
+        return $this;
+    }
```

Comment:
Following https://github.com/laravel/framework/pull/54067#discussion_r1901795250, this could be implemented like this (similar to https://github.com/laravel/framework/pull/54067/files#r1901780166):
```suggestion
    public function rfc(bool $strict = false)
    {
        if ($strict) {
            $this->strict = true;
        } else {
            $this->rfc = true;
        }

        return $this;
    }
```